### DISCUSSION TITLE
Please make SURVIVAL_VIDEO_URL, CREATIVE_VIDEO_URL, and SUBDIVISION_VIDEO_URL configurable

### DISCUSSION THREAD

**Discussion by DmitryRendov** [GriefPrevention/GriefPrevention#1721] [Ideas]
It is hardcoded in the code and I have to rebuild the code from my fork, instead of using the appropriate section in the config file.

https://github.com/TechFortress/GriefPrevention/blob/master/src/main/java/me/ryanhamshire/GriefPrevention/DataStore.java#L109-L111

Thanks!

**Comment by Jikoo** [comment]
You can already use your own video. In messages.yml, replace the replacement parameter (probably `{0}`, but haven't checked) with your URL.
